features/old/switches/branding.feature

```diff
@@ -13,7 +13,8 @@ Feature: Branding switch
     When I log in as provider "foo.example.com"
      And I go to the dns settings page
      And I follow "Change"
-    Then I should see the invitation to upgrade my plan
+    # FIXME: this is a false positive, the element is invisible
```

Comment:
Same as https://github.com/3scale/porta/pull/1143/files#r321244858
It can be the same issue 👍